extra-modules/extra-pluralkit/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/pluralkit/storage/PKGuildConfig.kt

```diff
@@ -22,9 +22,9 @@ import net.peanuuutz.tomlkt.TomlComment
 @Suppress("DataClassShouldBeImmutable", "MagicNumber")
 data class PKGuildConfig(
     @TomlComment(
-        "Base URL to use when attempting to hit the PluralKit API, without the /vX used to specify the version."
+        "Base URL to use when attempting to hit the PluralKit API."
```

Comment:
This change would entirely break the module for anyone using a custom API endpoint, and would also require them to update their URLs manually.

This is not a solution.